final_task/pycalc/arithmetic.py

```diff
@@ -0,0 +1,112 @@
+"""This module provides access to the arithmetic, comparison and auxiliary functions"""
+
+
+def sum_numbers(first_input=0, second_input=0):
+    """Get the sum of the numbers"""
+    return first_input + second_input
+
+
+def sub_numbers(first_input=0, second_input=0):
+    """Get the sub of the numbers"""
+    return first_input - second_input
+
+
+def division_number(first_input=1, second_input=1):
+    """Get the division of the numbers"""
+    return first_input / second_input
+
+
+def division_number_without_remainder(first_input=1, second_input=1):
+    """Get the division without remainder of the numbers"""
+    return first_input // second_input
+
+
+def modulus_division_number(first_input=1, second_input=1):
+    """Get the modulus of the numbers"""
+    return first_input % second_input
+
+
+def multiplication_number(first_input=1, second_input=1):
+    """Get the multiplication of the numbers"""
+    return first_input * second_input
+
+
+def less_or_equal(first_input=0, second_input=0):
+    """Return True if first number is less than or equal to second number"""
+    return first_input <= second_input
+
+
+def greater_or_equal(first_input=0, second_input=0):
+    """Return True if first number is greater than or equal to second number"""
+    return first_input >= second_input
+
+
+def less(first_input=0, second_input=0):
+    """Return True if first number is less than second number"""
+    return first_input < second_input
+
+
+def greater(first_input=0, second_input=0):
+    """Return True if first number is greater than second number"""
+    return first_input > second_input
+
+
+def equal(first_input=0, second_input=0):
+    """Return True if first number is equal to second number"""
+    return first_input == second_input
+
+
+def not_equal(first_input=0, second_input=0):
+    """Return True if first number is not equal to second number"""
+    return first_input != second_input
+
+
+OP_DICT = {
```

Comment:
NamedTuple would allow to combine function objects and their priorities in give excess to them by name